src/System.Transactions.Local/tests/NonMsdtcPromoterTests.cs

```diff
@@ -1585,7 +1587,7 @@ public static void TestCase_TransactionTimeout(bool promote)
             TestPassed();
         }
 
-        public static void TestCase_GetStatusAndDistributedId()
+        private static void TestCase_GetStatusAndDistributedId()
```

Comment:
I see that this one, which looks just like a `[Fact]` to me, is called by a `[Fact]` (which does nothing but call this).  I'll assume that is true for the rest of the void-no-arg public to private conversions in this file.

Reply:
Yeah, this doesn't look right but I'm not changing this as part of this PR.